source/includes/_payment.md

```diff
@@ -4109,11 +2720,7 @@ AuditLogs logs = paymentApi.getPaymentAttemptAuditLogsWithHistory(paymentAttempt
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-< Content-Type: application/json
-[
+```json
   {
     "changeType": "INSERT",
```

Comment:
The response should be an array, I think you missed adding [

Reply:
Hi Reshma, fixed all the places where array was required.